src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  onMovieAdd: (movie: Movie) => void,
+  isRepeat: boolean,
+  setRepeat: (ans: boolean) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({
+  onMovieAdd,
+  isRepeat,
+  setRepeat,
+}) => {
+  const [query, setQueries] = useState('');
+  const [error, setError] = useState(false);
+  const [isLoaded, setLoaded] = useState(false);
+  const [preview, setPreview] = useState<Movie | null>(null);
+
+  const findMovie = async () => {
+    const newMovie = await getMovie(query);
+
+    if ('Error' in newMovie) {
```

Comment:
Do you expect property with key 'Error' will be find in newMovie object? I don't think so

Reply:
If you do, anyway move checks on errors to request function